HelpDeskCommonMethods/src/main/java/webdriverbase/BaseTest.java

```diff
@@ -8,7 +8,7 @@
 public class BaseTest {
 	
     private WebDriver driver;
-
+//
```

Comment:
The Tester has mentioned the chrome driver Path Explicitly on line 15 under the BeforeSuite method
Try Using user.dir property to get the home directory location.
Or
You could use webdrivermanager from bonigarcia